net-utils/src/ip_echo_server.rs

```diff
@@ -29,8 +29,8 @@ impl IpEchoServerMessage {
         assert!(tcp_ports.len() <= msg.tcp_ports.len());
         assert!(udp_ports.len() <= msg.udp_ports.len());
 
-        msg.tcp_ports[..tcp_ports.len()].copy_from_slice(tcp_ports);
-        msg.udp_ports[..udp_ports.len()].copy_from_slice(udp_ports);
+        msg.tcp_ports[..].copy_from_slice(tcp_ports);
+        msg.udp_ports[..].copy_from_slice(udp_ports);
```

Comment:
@huitseeker , looks like this change causes some CI tests to fail with the panic `source slice length does not match destination slice length` because `msg.tcp_ports.len()` may be greater than `tcp_ports.len()` (ditto for udp_ports).
Am I missing something?

Reply:
🤦